01-map/extendUsersSettings.js

```diff
@@ -54,3 +54,17 @@ const USERS = [
     isEnabled: true
   },];
  */
+
+const extendUsersSettings = function (USERS) {
+  const userUpdate = USERS.map((element, index) => {
```

Comment:
`Array.prototype.map` returns an array (a collection). Better using plural.

```suggestion
  const users = USERS.map((element, index) => {
```